README.rst

```diff
@@ -666,7 +666,7 @@ Page Rendering
 GPU Rendering
 -------------
 
-* During the rendering process the graphical computing layers can use general
+* During the rendering, process the graphical computing layers can use general
```

Comment:
should the commer be after the process..!

~During the rendering, process the graphical computing layers can use general~
*During the rendering process, the graphical computing layers can use general*

but still, changes can be made based on the main document.